hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -168,12 +171,9 @@ def process_current_session(self, settings, parent_item):
                 document_item.expanded = False
                 document_item.checked = False
 
-            path = _document_path(document)
-
-            if path:
-                # try to set the thumbnail for display. won't display anything
-                # for psd/psb, but others should work.
-                document_item.set_thumbnail_from_path(path)
+            path = _document_path(document) 
+            if path: 
+                document_item.set_thumbnail_from_path(path) 
```

Comment:
Suggest adding the comment back in here. Should probably add it to line 124 as well.